src/App.vue

```diff
@@ -422,6 +397,51 @@ export default class App extends Mixins(DateMixin, FilingTemplateMixin, LegalApi
     }
   }
 
+  private async populateNameRequest (filing: any): Promise<void> {
+    try {
+      this.setCurrentDate(this.dateToUsableString(new Date()))
```

Comment:
This belongs somewhere else (ie, called always -- not only if we have a NR).

Reply:
turns out this is a duplicated line, and it is always called. removed

Reply:
I don't see where this is called elsewhere...?

Reply:
are you referring to "this.setCurrentDate(this.dateToUsableString(new Date()))" or to the method itself?

Reply:
Is setCurrentDate() called somewhere? I didn't see it.

If you're looking for a spot, may I recommend the top of fetchData()?

Reply:
that is where it's called...well, in the top of the try block in "fetchData"

Reply:
🤦 How did I miss this?